communication/include/communication/CommDefs.hpp

```diff
@@ -35,6 +36,7 @@ struct NodeInfo {
 };
 
 typedef std::unordered_map<NodeNum, NodeInfo> NodeMap;
+using namespace concord::secretsmanager;
```

Comment:
Don't put 'using namespace' in header files. It will pollute the namespace of every cpp using the header.